src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useCountdown.tsx

```diff
@@ -28,12 +28,6 @@ export const useCountdown = (
     })
   }, [onCompleted])
 
-  useEffect(() => {
-    return () => {
-      if (id.current) clearInterval(id.current)
-    }
-  }, [handleDecrement])
-
```

Comment:
btw this issue locally might due to 'use-strict' and the double render.

Reply:
> 🧠
> 
> [question]: How will we clear the interval on unmount? Not that this will cause a huuuge memory leak as this one is pretty tiny, but it will be there ticking away 😅

Oh very fair point... we wouldn't. And this is indeed only happening locally, so may be worth investigating some more before we get this in🙏🏽

Reply:
Big 🧠 about StrictMode ser - confirmed akschual build is happy! Reverted in bc317489db51d64f550e2b9cfaddeedc166d2884